src/python_testing/TC_OCC_3_2.py

```diff
@@ -110,35 +128,52 @@ async def test_TC_OCC_3_2(self):
         attrib_listener = ClusterAttributeChangeAccumulator(Clusters.Objects.OccupancySensing)
         await attrib_listener.start(dev_ctrl, node_id, endpoint=endpoint_id, min_interval_sec=0, max_interval_sec=30)
 
-        # TODO - Will add Namepiped to assimilate the manual sensor untrigger here
         self.step("3a")
-        self.wait_for_user_input(prompt_msg="Type any letter and press ENTER after DUT goes back to unoccupied state.")
+        # CI call to trigger off
+        if self.is_ci:
+            self.write_to_app_pipe('{"Name":"SetOccupancy", "EndpointId": 1, "Occupancy": 0}')
+        else:
+            self.wait_for_user_input(prompt_msg="Type any letter and press ENTER after DUT goes back to unoccupied state.")
 
         self.step("3b")
         if attributes.Occupancy.attribute_id in attribute_list:
             initial_dut = await self.read_occ_attribute_expect_success(attribute=attributes.Occupancy)
             asserts.assert_equal(initial_dut, 0, "Occupancy attribute is still detected state")
 
-        # TODO - Will add Namepiped to assimilate the manual sensor trigger here
         self.step("3c")
-        self.wait_for_user_input(
-            prompt_msg="Type any letter and press ENTER after the sensor occupancy is triggered and its occupancy state changed.")
+        # CI call to trigger on
+        if self.is_ci:
+            self.write_to_app_pipe('{"Name":"SetOccupancy", "EndpointId": 1, "Occupancy": 1}')
+            initial_dut = await self.read_occ_attribute_expect_success(attribute=attributes.Occupancy)
+            time.sleep(2)
+            # will write one more to consier priming report effect of the previous CI writing
+            self.write_to_app_pipe('{"Name":"SetOccupancy", "EndpointId": 1, "Occupancy": 0}')
+            diff_val = await self.read_occ_attribute_expect_success(attribute=attributes.Occupancy)

```

Comment:
This makes too many low-level assumptions. The CI should only replace prompts with single individual action equivalent to the prompt. This is separated test logic so may have separate bugs